<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DemonstrationWindow</class>
 <widget class="QWidget" name="DemonstrationWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>969</width>
    <height>540</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>708</width>
    <height>540</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>969</width>
    <height>540</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Particles in Box</string>
  </property>
  <widget class="QSlider" name="current_state">
   <property name="geometry">
    <rect>
     <x>4</x>
     <y>454</y>
     <width>700</width>
     <height>20</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QPushButton" name="button_play">
   <property name="geometry">
    <rect>
     <x>336</x>
     <y>490</y>
     <width>32</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>▯▯</string>
   </property>
  </widget>
  <widget class="QPushButton" name="button_backward">
   <property name="geometry">
    <rect>
     <x>294</x>
     <y>490</y>
     <width>32</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>⧏</string>
   </property>
  </widget>
  <widget class="QPushButton" name="button_forward">
   <property name="geometry">
    <rect>
     <x>378</x>
     <y>490</y>
     <width>32</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>⧐</string>
   </property>
  </widget>
  <widget class="QFrame" name="frame_player">
   <property name="geometry">
    <rect>
     <x>4</x>
     <y>4</y>
     <width>700</width>
     <height>450</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Plain</enum>
   </property>
  </widget>
  <widget class="QSpinBox" name="fps">
   <property name="geometry">
    <rect>
     <x>656</x>
     <y>490</y>
     <width>48</width>
     <height>32</height>
    </rect>
   </property>
   <property name="wrapping">
    <bool>false</bool>
   </property>
   <property name="minimum">
    <number>1</number>
   </property>
   <property name="maximum">
    <number>60</number>
   </property>
   <property name="value">
    <number>30</number>
   </property>
  </widget>
  <widget class="QLabel" name="label_fps">
   <property name="geometry">
    <rect>
     <x>502</x>
     <y>490</y>
     <width>146</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>FPS</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="PlotWidget" name="plot_maxwell">
   <property name="geometry">
    <rect>
     <x>708</x>
     <y>4</y>
     <width>255</width>
     <height>255</height>
    </rect>
   </property>
  </widget>
  <widget class="PlotWidget" name="plot_boltzmann">
   <property name="geometry">
    <rect>
     <x>708</x>
     <y>265</y>
     <width>255</width>
     <height>255</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>4</x>
     <y>490</y>
     <width>221</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>Time elapsed: {time:.5f}s</string>
   </property>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>PlotWidget</class>
   <extends>QGraphicsView</extends>
   <header location="global">pyqtgraph</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
